akka-persistence/src/test/scala/akka/persistence/PersistentActorStashingSpec.scala

```diff
@@ -166,7 +166,7 @@ abstract class PersistentActorStashingSpec(config: Config) extends PersistenceSp
 
       cmds foreach (persistentActor ! _)
       persistentActor ! GetState
-      expectMsg(evts)
+      expectMsg(evts.toList)
```

Comment:
I have not changed scala version in this PR. Failure comes from both Jenkins and my machine. I have no clue why.